.github/workflows/ci.yml

```diff
@@ -8,6 +8,9 @@ on:
     branches:
       - master
 
+env:
+  GOPROXY: https://proxy.golang.org/
```

Comment:
is this not set by default?

Reply:
The default (according to `go env`) is `https://proxy.golang.org,direct` and so the idea here is that by excluding `direct` we can effectively protect ourselves against the "leftpad situation" where the source would suddenly disappear by making sure that all dependencies are always cached before being added.